### PR TITLE
Add page nav and title anchors

### PR DESCRIPTION
**Pull request by mosabua**
- optional feature per page
- used in some community pages

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -6,7 +6,27 @@
...
+{% if page.show_pagenav  %}
+<div class="row" style="margin-left:0; margin-right:0;">
+  <div class="col-md-2 col-sm-12">
+  </div>
+  <div class="col-md-8 col-sm-12">
+    <div class="mx-2">
+      {% include anchor_headings.html html=content anchorBody="#" %}
+    </div>
+  </div>
+  <div class="col-md-2 d-md-block d-none">
```
Comment (colebow) [inline _layouts/default.html:19]: We need to make sure our TOC is getting more width for medium screens, with only md-2 it gets unsightly if you make things narrower:
image

You can see what I settled on for the Blog TOC in the blog.html file.

**Review by colebow — CHANGES_REQUESTED** [review]
LGTM % fix for smaller screens.

File: _layouts/default.html (hunk 1 above)
Comment (mosabua) [reply _layouts/default.html:19]: Lets debug together in our meeting later ... using md-3 just didnt look so good.. there was LOTS of additional space.

File: _layouts/default.html [hunk 2]
```diff
@@ -6,7 +6,27 @@
 {%- include hero.html -%}
 {%- endif -%}
 
-{{ content }}
+  <!-- Comment to remove TOC from a page -->
+{% if page.show_pagenav  %}
+<div class="row" style="margin-left:0; margin-right:0;">
+  <div class="col-md-2 col-sm-12">
+  </div>
+  <div class="col-md-8 col-sm-12">
```
Comment (colebow) [inline _layouts/default.html:14]:
```suggestion
  <div class="col-xl-8 col-sm-12">
```

File: _layouts/default.html (hunk 1 above)
Comment (colebow) [inline _layouts/default.html:19]:
```suggestion
  <div class="col-xl-2 d-xl-block d-none">
```

**Review by colebow — COMMENTED** [review]
If we think it's important to not waste space, this is only going to work for big screens, so let's keep it to them.

**Pull request merged**